datasource.go

```diff
@@ -0,0 +1,120 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"strconv"
+
+	"golang.org/x/net/context"
+
+	"github.com/grafana/grafana-plugin-model/go/datasource"
+	"github.com/grafana/grafana/pkg/components/simplejson"
```

Comment:
could we replace simplejson from grafana/grafana
to https://github.com/bitly/go-simplejson ? `grafana/grafana` doesn't look a library code